terraform/bod_docker_ec2.tf

```diff
@@ -174,6 +175,7 @@ module "bod_docker_ansible_provisioner" {
     # This file will be used to add/override any settings in
     # docker-compose.yml (for cyhy-mailer).
     "docker_compose_override_file_for_mailer=${var.docker_mailer_override_filename}",
+    "ses_send_email_role=arn:aws:iam::${var.ses_account_id}:role/SesSendEmail-cyber.dhs.gov",
```

Comment:
Am I missing something?  Shouldn't this be:
```"ses_send_email_role={var.ses_role_arn}"```

Reply:
Fixed with a dramatic force push to remove the role name from the history.  See commit fdefaf7.

Reply:
👍